[FR] Update `Beatmap/Beatmap description` and `History of Loved` (#5842)
ppy/osu-wiki: wiki/Beatmap/History_of_Loved/fr.md

# L'histoire de la catégorie Loved

> La catégorie [Loved](/wiki/Beatmap/Category#loved) d'une beatmap est actuellement gérée par le [Project Loved](/wiki/Project_Loved). Cependant, cela n'a pas toujours été le cas. Project Loved est le troisième système principal qui a promu les beatmaps au rang de Loved ; avant lui, il y avait d'autres méthodes pour le faire, ainsi que de longues périodes de stagnation pour la catégorie Loved.

Reviewer: ```suggestion
La catégorie de beatmaps [Loved](/wiki/Beatmap/Category#loved) est actuellement gérée par le [Project Loved](/wiki/Project_Loved). Cependant, cela n'a pas toujours été le cas. Project Loved est le troisième système principal qui a promu les beatmaps au rang de Loved ; avant lui, il y avait d'autres méthodes pour le faire, ainsi que de longues périodes de stagnation pour la catégorie Loved.
```

## Demande de fonctionnalité (février 2015)

> Le 02/02/2015, la demande de fonctionnalité *[New beatmap category for good unrankable mapsets](https://osu.ppy.sh/community/forums/topics/293841)* a été postée. Elle détaillait un autre moyen en dehors du [processus de classement](/wiki/Beatmap_ranking_procedure) pour que les maps puissent avoir des classements, et a rapidement gagné en popularité car les joueurs ont montré leur inquiétude quant au processus de classement devenant trop strict et ne permettant pas à certaines maps d'abriter la compétition.

Reviewer: ```suggestion
Le 02/02/2015, la demande de fonctionnalité *[New beatmap category for good unrankable mapsets](https://osu.ppy.sh/community/forums/topics/293841)* a été postée. Elle détaillait un autre moyen en dehors du [processus de classement](/wiki/Beatmap_ranking_procedure) pour que les maps puissent avoir des classements, et a rapidement gagné en popularité car les joueurs ont montré leur inquiétude quant au processus de classement devenant trop strict et ne permettant pas à certaines maps de prendre part à la compétition.
```

> Une autre raison pour laquelle cette demande est devenue populaire est que des maps très peu conventionnelles ou révolutionnaires n'auraient jamais pu correspondre aux [critères de classement](/wiki/Ranking_Criteria). Les joueurs étaient intéressés par la compétition sur certaines de ces maps, comme celles présentant un style "2B", un jeu de type *[Aspire](/wiki/Contests/Aspire)* ou des éléments de [TAG co-op](/wiki/Beatmap/TAG).

Reviewer: ```suggestion
Une autre raison pour laquelle cette demande est devenue populaire est que des maps très peu conventionnelles ou révolutionnaires n'auraient jamais pu correspondre aux [critères de classement](/wiki/Ranking_Criteria). Les joueurs étaient intéressés par la compétition sur certaines de ces maps, comme celles présentant un style "2B", un gameplay de type *[Aspire](/wiki/Contests/Aspire)* ou des éléments de [TAG co-op](/wiki/Beatmap/TAG).
```

> Fin 2016, cette demande de fonctionnalité détenait le deuxième plus grand nombre d'étoiles dans l'histoire du forum, avec près de 13 000 priorités.

Reviewer: ```suggestion
Fin 2016, cette demande de fonctionnalité détenait le deuxième plus grand nombre d'étoiles dans l'histoire du forum, avec près de 13 000 de priorité.
```

## Vote communautaire (Octobre 2016 - Janvier 2017)

> Lorsque la catégorie [Loved a été créé](https://osu.ppy.sh/home/news/2016-10-17-here-comes-the-love), il visait à répondre à toutes les demandes de fonctionnalités. Des sondages ont été ouverts par ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) sur les forums où chacun pouvait aider à décider quelles maps deviendraient Loved. Les détails du système de vote ont été expliqués dans le message d'actualité *[Show Your Love !](https://osu.ppy.sh/home/news/2016-10-21-show-your-love)*.

Reviewer: ```suggestion
Lorsque [la catégorie Loved a été créée](https://osu.ppy.sh/home/news/2016-10-17-here-comes-the-love), il a fallu répondre à toutes les questions concernant cette demande de fonctionnalité. Des sondages ont été ouverts par ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) sur les forums où chacun pouvait aider à décider quelles maps deviendraient Loved. Les détails du système de vote ont été expliqués dans le newspost *[Show Your Love !](https://osu.ppy.sh/home/news/2016-10-21-show-your-love)*.
```

> Les premiers tours de scrutin ont connu un grand succès, la communauté ayant pris plaisir à participer à cette nouvelle fonctionnalité. Malheureusement, il n'a pas fallu longtemps pour que certaines préoccupations importantes et difficiles à résoudre soient mises en lumière. À l'issue du [quatrième tour de scrutin](https://osu.ppy.sh/community/forums/topics/541452), les membres de la communauté se sont plaints que, même s'ils pouvaient voter, ils n'avaient pas l'impression d'avoir beaucoup d'influence sur le choix des maps qui étaient soumises au vote. Beaucoup ont mentionné que la rareté des votes ne soutenait pas l'idée de Loved. Il était également préoccupant que les maps soient effectivement en concurrence les unes avec les autres lors de ce vote, bien que toutes les maps aient été populaires d'une manière ou d'une autre, et donc toutes apparemment adaptées à Loved.

Reviewer: ```suggestion
Les premiers tours de scrutin ont connu un grand succès, la communauté ayant pris plaisir à participer à cette nouvelle fonctionnalité. Malheureusement, il n'a pas fallu longtemps pour que certains problèmes importants et difficiles à résoudre surviennent. À l'issue du [quatrième tour de scrutin](https://osu.ppy.sh/community/forums/topics/541452), les membres de la communauté se sont plaints que, même s'ils pouvaient voter, ils n'avaient pas l'impression d'avoir beaucoup d'influence sur le choix des maps qui étaient soumises au vote. Beaucoup ont mentionné que la rareté des votes ne soutenait pas l'idée de Loved. Il était également préoccupant que les maps soient effectivement en concurrence les unes avec les autres lors de ce vote, bien que toutes les maps aient été populaires d'une manière ou d'une autre, et donc toutes apparemment adaptées à la catégorie Loved.
```

> Suite aux commentaires reçus lors du quatrième tour et dans [un sondage général sur l'état de Loved](https://osu.ppy.sh/community/forums/topics/529407) ouvert environ un mois avant celui-ci, Ephemeral (et d'autres membres de l'équipe d'osu!) a décidé que les votes dans ce format ne seraient pas viables à long terme. Il a fortement soutenu l'idée d'utiliser le système de favoris comme données afin de décider quelles maps seraient admises dans la catégorie Loved.

Reviewer: ```suggestion
Suite aux commentaires reçus lors du quatrième tour et dans [un sondage général sur l'état de la catégorie Loved](https://osu.ppy.sh/community/forums/topics/529407) ouvert environ un mois avant celui-ci, Ephemeral (et d'autres membres de l'équipe d'osu!) a décidé que les votes dans ce format ne seraient pas viables à long terme. Il a fortement soutenu l'idée d'utiliser le système de beatmaps favorites comme mesure afin de décider quelles maps seraient admises dans la catégorie Loved.
```

## Star priority et favoris (Janvier 2017 - Juin 2017)

> Quelques semaines seulement après le quatrième tour de scrutin de la communauté, ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) a publié [une annonce](https://osu.ppy.sh/community/forums/topics/549835) décrivant le prochain système d'admission des beatmaps dans la catégorie Loved. Au lieu de mettre une sélection de maps dans les sondages, chaque map devient éligible pour la Loved, tant qu'elle a au moins 30 favoris et 100 [star priority (SP)](/wiki/Modding/Star_priority).

Reviewer: ```suggestion
Quelques semaines seulement après le quatrième tour de scrutin de la communauté, ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) a publié [une annonce](https://osu.ppy.sh/community/forums/topics/549835) décrivant le prochain système d'admission des beatmaps dans la catégorie Loved. Au lieu de mettre une sélection de maps dans les sondages, toutes les maps deviennent éligible à la catégorie Loved, tant qu'elle a au moins 30 favoris et 100 [star priority (SP)](/wiki/Modding/Star_priority).
```

> Quelques semaines seulement après le quatrième tour de scrutin de la communauté, ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) a publié [une annonce](https://osu.ppy.sh/community/forums/topics/549835) décrivant le prochain système d'admission des beatmaps dans la catégorie Loved. Au lieu de mettre une sélection de maps dans les sondages, toutes les maps deviennent éligible à la catégorie Loved, tant qu'elle a au moins 30 favoris et 100 [star priority (SP)](/wiki/Modding/Star_priority).

Reviewer: ```suggestion
Quelques semaines seulement après le quatrième tour de scrutin de la communauté, ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) a publié [une annonce](https://osu.ppy.sh/community/forums/topics/549835) décrivant le prochain système d'admission des beatmaps dans la catégorie Loved. Au lieu de mettre une sélection de maps dans les sondages, toutes les maps deviennent éligible à la catégorie Loved, à condition qu'elles aient au moins 30 favoris et 100 [star priority (SP)](/wiki/Modding/Star_priority).
```

> L'exigence d'un SP élevé impliquait que la communauté de mapping et de modding soit impliquée dans le processus de transfert des maps vers Loved : Le SP ne pouvait être ajouté aux maps qu'en dépensant du [kudosu!](/wiki/Modding/Kudosu), une monnaie obtenue en moddant d'autres maps. Ce point est immédiatement devenu un sujet de controverse avec la communauté au sens large, qui pensait que la catégorie Loved était censé permettre d'obtenir des classements sur les maps populaires, et pas seulement sur celles que les "mappeurs" aimaient.

Reviewer: ```suggestion
L'exigence d'une SP élevée impliquait que la communauté de mapping et de modding soit impliquée dans le processus de transfert des maps vers la catégorie Loved : La SP ne pouvait être ajoutée aux maps qu'en dépensant des [kudosu!](/wiki/Modding/Kudosu), une monnaie obtenue en moddant d'autres maps. Ce point est immédiatement devenu un sujet de controverse avec la communauté au sens large, qui pensait que la catégorie Loved était censée permettre aux beatmaps populaires d'obtenir un classement, et pas seulement celles que les "mappeurs" aimaient.
```

> Au cours de la période d'application de ce système, certaines des maps nouvellement Loved sont devenues des sujets de controverse, les joueurs affirmant que c'était exactement ce qu'ils craignaient, à savoir utiliser une grande quantité de SP et un nombre relativement faible de favoris pour faire Loved une map. D'un autre côté, la minorité qui appréciait ce système se plaignait également du fait que le nombre de SP requis était absurdement élevé : obtenir 100 SP demandait un tel investissement en temps et en efforts que cela ne valait pas la peine de contribuer au cycle des maps Loved. Ces opinions et bien d'autres se trouvent dans le fil de discussion de la [lettre ouverte à la communauté d'Ephemeral](https://osu.ppy.sh/community/forums/topics/601062) qui, espérait-il, fournirait les informations nécessaires pour résoudre les problèmes apparents de Loved.

Reviewer: ```suggestion
Au cours de la période d'application de ce système, certaines des maps nouvellement Loved sont devenues des sujets de controverse, les joueurs affirmant que c'était exactement ce qu'ils craignaient, à savoir utiliser une grande quantité de SP et un nombre relativement faible de favoris pour qu'une map soit Loved. D'un autre côté, la minorité qui appréciait ce système se plaignait également du fait que le nombre de SP requis était absurdement élevé : obtenir 100 SP demandait un tel investissement en temps et en efforts que cela ne valait pas la peine de contribuer au cycle des maps Loved. Ces opinions et bien d'autres se trouvent dans le fil de discussion de la [lettre ouverte à la communauté d'Ephemeral](https://osu.ppy.sh/community/forums/topics/601062) qui, espérait-il, fournirait les informations nécessaires pour résoudre les problèmes apparents de Loved.
```

Ephemeral a posté quelques mises à jour plus tard dans son fil de discussion sur le forum, mais aucune mesure n'a été prise et Loved est devenu inactif au cours du mois de juin 2017.

> ## Capitaine pick et projet Loved (Septembre 2017 - le présent)

Reviewer: ```suggestion
## Captain's pick et Project Loved (Septembre 2017 - présent)
```

> Le [deuxième épisode du talk-show *Coffee Hour* d'osu!](https://www.youtube.com/watch?v=JYLITcBYkC4) a mis en vedette ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802), ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335), et ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) discutant du passé et du futur de Loved. Vers la fin, Ephemeral a suggéré que le vote original de la communauté pourrait être ramené sous une forme ou une autre, puisque la communauté semblait l'apprécier davantage que le système de priorités et de favoris par étoiles, à condition que la sélection des maps soit bonne. La question de savoir comment choisir ces maps est restée sans réponse dans cette émission.

Reviewer: ```suggestion
Le [deuxième épisode du talk-show *Coffee Hour* d'osu!](https://www.youtube.com/watch?v=JYLITcBYkC4) était présenté par ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802), ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335), et ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) discutant du passé et du futur de Loved. Vers la fin, Ephemeral a suggéré que le vote original de la communauté pourrait être ramené sous une forme ou une autre, puisque la communauté semblait l'apprécier davantage que le système de Star Priority et de beatmaps favorites, à condition que la sélection des maps soit bonne. La question de savoir comment choisir ces maps est restée sans réponse durant cette émission.
```

Fin septembre, ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) a collaboré avec Ephemeral pour créer une nouvelle version du vote communautaire appelée [Loved Captain's Pick](https://osu.ppy.sh/home/news/2017-09-25-introducing-the-loved-captains-pick). Au début, il était presque identique à l'ancien système de vote, à l'exception du fait que les maps des sondages étaient choisies par les "capitaines", des membres de confiance de la communauté qui avaient une bonne idée de ce que le reste de la communauté voulait voir.

> Avant le deuxième tour du "Captain's Pick", Toy [a annoncé sur les forums](https://osu.ppy.sh/community/forums/topics/662761) que les maps choisies pour le vote proviendraient des soumissions de la communauté sur un [formulaire](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform) et une [feuille](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit) Google. Toy espérait que cela "fonctionnerait mieux à long terme que les précédentes mises en œuvre de Loved" en donnant aux participants de Loved plus de poids dans le vote.

Reviewer: ```suggestion
Avant le deuxième tour du "Captain's Pick", Toy [a annoncé sur les forums](https://osu.ppy.sh/community/forums/topics/662761) que les maps choisies pour le vote proviendraient des soumissions de la communauté sur un [formulaire](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform) et une [feuille](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit) Google. Toy espérait que cela "fonctionnerait mieux à long terme que les précédentes mises en œuvre de Loved" en donnant aux participants du Project Loved plus de poids dans le vote.
```

> Avant le deuxième tour du "Captain's Pick", Toy [a annoncé sur les forums](https://osu.ppy.sh/community/forums/topics/662761) que les maps choisies pour le vote proviendraient des soumissions de la communauté sur un [formulaire](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform) et une [feuille](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit) Google. Toy espérait que cela "fonctionnerait mieux à long terme que les précédentes mises en œuvre de Loved" en donnant aux participants du Project Loved plus de poids dans le vote.

Reviewer: ```suggestion
Avant le deuxième tour du "Captain's Pick", Toy [a annoncé sur les forums](https://osu.ppy.sh/community/forums/topics/662761) que les maps choisies pour le vote proviendraient des soumissions de la communauté sur un [formulaire](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform) et une [spreadsheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit) Google. Toy espérait que cela "fonctionnerait mieux à long terme que les précédentes mises en œuvre de Loved" en donnant aux participants du Project Loved plus de poids dans le vote.
```

> Après quelques tours du style de sondage original, Toy a décidé d'utiliser un format de vote légèrement différent où les maps ne seraient plus en compétition les unes avec les autres. À partir de ce moment, chaque map a eu son propre sondage, demandant à la communauté si elle voulait ou non que cette map spécifique soit ajouté a la catégorie Loved. Si un pourcentage minimum de votants répondait "oui", la map passait le cap du vote. En raison du nombre élevé de sondages sur le forum, un [nouveau forum](https://osu.ppy.sh/community/forums/120) a été créé spécifiquement pour le vote sur les maps Loved, et c'est à partir de ce moment que le projet a été officiellement appelé "Project Loved".

Reviewer: ```suggestion
Après quelques tours du style de sondage originel, Toy a décidé d'utiliser un format de vote légèrement différent où les maps ne seraient plus en compétition les unes avec les autres. À partir de ce moment, chaque map a eu son propre sondage, demandant à la communauté si elle voulait ou non que cette map spécifique soit ajoutée à la catégorie Loved. Si un pourcentage minimum de votants répondait "oui", la map passait le cap du vote. En raison du nombre élevé de sondages sur le forum, un [nouveau forum](https://osu.ppy.sh/community/forums/120) a été créé spécifiquement pour le vote des potentielles maps Loved, et c'est à partir de ce moment que le projet a été officiellement appelé "Project Loved".
```

> ## Le présent

Reviewer: ```suggestion
## Système actuel
```

> Le [Project Loved](/wiki/Project_Loved) est le système actuel utilisé pour transférer les maps vers la catégorie Loved. Vous trouverez plus de détails et d'informations sur la façon d'obtenir des maps pour le vote dans le Project Loved dans son article wiki.   

Reviewer: ```suggestion
Le [Project Loved](/wiki/Project_Loved) est le système actuel utilisé pour transférer les maps vers la catégorie Loved. Vous trouverez plus de détails et d'informations sur comment rendre des beatmaps éligibles aux votes dans l'article wiki du Project Loved.   
```
